GitUI/CommandsDialogs/BrowseDialog/FormBisect.cs

```diff
@@ -18,22 +18,19 @@ public sealed partial class FormBisect : GitModuleForm
         private readonly RevisionGridControl _revisionGrid;
 
         [Obsolete("For VS designer and translation test only. Do not remove.")]
+#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
         private FormBisect()
+#pragma warning restore CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
         {
             InitializeComponent();
         }
```

Comment:
We have several forms/controls where a private constructor is added only for the designer/translator. Because these ctors don't assign all fields with non-null values, we'd usually see CS8618. In each case I suppress the diagnostic.